tsconfig.json

```diff
@@ -1,16 +1,17 @@
 {
   "compilerOptions": {
-    "target": "ES6",
-    "allowSyntheticDefaultImports": true,
-    "experimentalDecorators": true,
-    "esModuleInterop": true,
-    "outDir": "build",
-    "forceConsistentCasingInFileNames": true,
     "strict": true,
-    "skipLibCheck": true,
-    "moduleResolution": "Node16",
-    "module": "Node16"
+    "esModuleInterop": true,
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
+    "target": "ES6",
+    "baseUrl": "./",
+    "paths": {
+      "@src/*": ["src/*"],
+      "@lib/*": ["src/lib/*"],
+      "@middleware/*": ["src/middleware/*"],
+      "@routes/*": ["src/routes/*"]
+    }
   },
-  "include": ["src/**/*.ts"],
-  "exclude": ["node_modules", "test/**/*.ts"]
+  "include": ["src/**/*.ts", "test/**/*.ts"]
```

Comment:
Included tests files so that we can use shortened paths (defined above) in there. This is the change that led to having an extra `tsconfig.build.json` file to make sure they are excluded during build time.